src/fondue/Fondue.js

```diff
@@ -188,28 +188,33 @@ export default class Fondue {
 	// Usage:
 	//   fondue.features
 	get features() {
+		const featureResult = {};
 		const result = this._raw("GSUB");
```

Comment:
What do you think about naming this `gsub`? `result` is very generic, naming it more specifically is clearer imo.

Reply:
I'm about to dig into this code soon and will probably refactor this quite a bit, so let's leave it until then. Works good enough for now :-)